packages/core/upload/admin/src/hooks/useBulkRemove.js

```diff
@@ -24,7 +23,7 @@ export const useBulkRemove = () => {
       return acc;
     }, {});
 
-    return post(url, payload);
```

Comment:
In these upload plugin cases do you think it's worth using `pluginId` ?

Reply:
No - that wouldn't make the situation any better, because we still couldn't grep for it. I don't see us changing the name of one of these plugins anytime soon and even if we'd probably run into a whole lot of different problems other than replacing the API endpoints 😓

Reply:
I don't think there's a huge benefit, I think realistically it would _potentially_ make more sense to have a dictionary of URLs

Reply:
I was actually worried about the size impact this would have, but turns out gzip just eats up these strings 🚀